[WIP] Update Chapter - Research Community Management (#3684)
the-turing-way/the-turing-way: book/website/collaboration/research-infrastructure-roles/community-manager.md

(cl-infrastructure-community-managers)=
# Research Community Managers: Overview

> **test for Jessie to delete!!**

Reviewer: ```suggestion

```

> Community Management as a profession was developed to facilitate the social, informal and formal learning processes, which take place in Communities of Practice - "groups of people who share a concern, a set of problems, or a passion about a topic, and who deepen their knowledge and expertise in this area by interacting on an ongoing basis" @Lave1991situatedlearning. 

Reviewer 2: ```suggestion
Community Management as a profession was developed to facilitate the social, informal and formal learning processes, which take place in Communities of Practice - "groups of people who share a concern, a set of problems, or a passion about a topic, and who deepen their knowledge and expertise in this area by interacting on an ongoing basis" @lave1991situatedlearning. 
```

> Community Management as a profession was developed to facilitate the social, informal and formal learning process which take place in Communities of Practice - "groups of people who share a concern, a set of problems, or a passion about a topic, and who deepen their knowledge and expertise in this area by interacting on an ongoing basis" <mark>Ref: Lave, Jean, and Etienne Wenger. Situated Learning: Legitimate Peripheral Participation. Cambridge, Cambridge University Press, 1991.</mark>. 

Reviewer 2: ```suggestion
Community Management as a profession was developed to facilitate the social, informal and formal learning processes, which take place in Communities of Practice - "groups of people who share a concern, a set of problems, or a passion about a topic, and who deepen their knowledge and expertise in this area by interacting on an ongoing basis" <mark>Ref: Lave, Jean, and Etienne Wenger. Situated Learning: Legitimate Peripheral Participation. Cambridge, Cambridge University Press, 1991.</mark>. 
```

> Community Management as a profession was developed to facilitate the social, informal and formal learning processes, which take place in Communities of Practice - "groups of people who share a concern, a set of problems, or a passion about a topic, and who deepen their knowledge and expertise in this area by interacting on an ongoing basis" <mark>Ref: Lave, Jean, and Etienne Wenger. Situated Learning: Legitimate Peripheral Participation. Cambridge, Cambridge University Press, 1991.</mark>. 

Reviewer 2: ```suggestion
Community Management as a profession was developed to facilitate the social, informal and formal learning processes, which take place in Communities of Practice - "groups of people who share a concern, a set of problems, or a passion about a topic, and who deepen their knowledge and expertise in this area by interacting on an ongoing basis" @Lave1991situatedlearning. 
```

> Community Managers have been employed in open source projects in technical industries  since the early 1990's @Michlmayr2009community , and more recently in research organisations (Universities and research institutes), where they are often known as Research Community Managers.

Reviewer 2: ```suggestion
Community Managers have been employed in open source projects in technical industries  since the early 1990's @michlmayr2009community , and more recently in research organisations (Universities and research institutes), where they are often known as Research Community Managers.
```

> Community Managers have been employed in open source projects in technical industries  since the early 1990's <mark>Ref:  Michlmayr, M., 2009. Community management in open source projects. The European Journal for the Informatics Professional, X (3), pp.22-26. https://www.cyrius.com/publications/michlmayr-community_management.pdf</mark>, and more recently in research organisations (Universities and research institutes, where they are known as Research Community Managers.

Reviewer 2: ```suggestion
Community Managers have been employed in open source projects in technical industries  since the early 1990's <mark>Ref:  Michlmayr, M., 2009. Community management in open source projects. The European Journal for the Informatics Professional, X (3), pp.22-26. https://www.cyrius.com/publications/michlmayr-community_management.pdf</mark>, and more recently in research organisations (Universities and research institutes), where they are often known as Research Community Managers.
```

> Community Managers have been employed in open source projects in technical industries  since the early 1990's <mark>Ref:  Michlmayr, M., 2009. Community management in open source projects. The European Journal for the Informatics Professional, X (3), pp.22-26. https://www.cyrius.com/publications/michlmayr-community_management.pdf</mark>, and more recently in research organisations (Universities and research institutes), where they are often known as Research Community Managers.

Reviewer 2: ```suggestion
Community Managers have been employed in open source projects in technical industries  since the early 1990's @Michlmayr2009community , and more recently in research organisations (Universities and research institutes), where they are often known as Research Community Managers.
```

```

(cl-infrastructure-community-managers-tasks)=
## What do Research Community Managers do? 

Research Community Managers work to engage, enthuse and organise groups of scientists, researchers and/or patients and the public around shared research topics and objectives. 
They are often involved in culture change to support collaboration between members of the community, through the development of community leadership and agency. 
They value the people doing the research, and develop structures to support the generation of bottom-up activity <mark>does bottom-up need a definition/glossary listing?</mark>. Their goals are to:

* Embed open, inclusive and reproducible research practices
* Ensure a shared understanding of goals, roadmap, processes
* Facilitate stakeholder engagement and collaboration
* Provide technical support and domain expertise
* Co-create, maintain and communicate project resources
* Amplify and champion community learnings and achievements

> Each role is varied but the main activities are typically focused around advocacy for the community, stewardship, scaffolding for collaboration, and the flow of information across all levels of the research, both to and from leadership. and collaboration with the community. 

Reviewer 2: ```suggestion
Each role is varied but the main activities are typically focused around advocacy for the community, stewardship, scaffolding for collaboration, and the flow of information across all levels of the research, both to and from leadership, and collaboration with the community. 
```

> Effective collaboration can also require management of relationships between colleagues, including coaching, formal and informal mediation, reflexive exercises, and dealing with formal Code of Conduct <mark>link to glossary?</mark>. 

Reviewer 2: ```suggestion
Effective collaboration can also require management of relationships between colleagues, including coaching, formal and informal mediation, reflexive exercises, and dealing with formal Code of Conduct <mark>link to glossary?</mark> processes. 
```

> The aim of this work is to understand and gather learnings about how and where teams can work well together and diagnose issues which have contributed to challenges in collaboration. The Research Community Manger should then be well positioned to develop systems to support effective and inclusive collaboration, and mitigate harm to individuals or groups of the community. 

Reviewer 2: ```suggestion
The aim of this work is to understand and gather learnings about how and where teams can work well together and diagnose issues, that have contributed to challenges in collaboration. The Research Community Manager should then be well positioned to develop systems to support effective and inclusive collaboration, and mitigate harm to individuals or groups of the community. 
```

Themes of Research Community Manager work can be:
* Catalysing connections with all stakeholders
* Fostering relationships within and between communities (integrating communities together)
* Linking the right people up together (based on expertise, interests and project goals)

> * Identifying and growing engaged community members into community leaders

Reviewer: ```suggestion
* Identifying and developing engaged community members into community leaders
```

> * Identifying and growing engaged community members into community leaders

Reviewer: Identifying and developing engaged community members into community leaders

> * Bridging between technical and non-technical members of the community by building and delivering training

Reviewer: ```suggestion
* Bridging between technical and non-technical members of the community by creating and delivering training
```

> * Bridging between technical and non-technical members of the community by building and delivering training

Reviewer: * Bridging between technical and non-technical members of the community by **creating** and delivering training
```

> * Encouraging high standards of reproducible, ethical, inclusive and collaborative data science

Reviewer 2: ```suggestion
* Encouraging, modelling and mentoring community members in high standards of reproducible, ethical, inclusive and collaborative data science
```

> * Facilitating the creation of sub-networks within the community around shared experiences, for example, such as an early career researcher network

Reviewer: (remove 'such as')
* Facilitating the creation of sub-networks within the community around shared experiences, for example, an early career researcher network

> The day-to-day tasks of a community manager could include: 

Reviewer 2: ```suggestion
The day-to-day tasks of a Research Community Manager could include: 
```

> * Developing a strategy for community activities, in alignment with the goals of the wider research project

Reviewer: remove coma after activities
```suggestion
* Developing a strategy for community activities in alignment with the goals of the wider research project
```

* Organising and hosting community calls
* Onboarding new members
* Attending community and project leadership meetings
* Writing community reports or newsletters

> * Maintaining and updating the community site 

Reviewer 2: ```suggestion
* Maintaining and updating the community repositories and online spaces
* Collaboratively writing academic papers
```

> * Maintaining and catalysing communication on community fora 

Reviewer 2: ```suggestion
* Maintaining and catalysing communication on community forums
```

* Planning for upcoming community initiatives

> * Running training courses & workshops

Reviewer: ```suggestion
* Running training courses and workshops
```

* Reading and sharing publications relevant to the community
* Managing membership lists

> * Mentoring community contributions

Reviewer: ```suggestion
* Mentoring community contributors
```

> * Meeting and listening to community members to understand their needs and challenges

Reviewer: ```suggestion
* Meeting with and listening to community members to understand their needs and challenges
```

* Maintaining stakeholder and project databases

> * Advocating for the community and project at external national and international meetings

Reviewer 2: ```suggestion
* Advocating for the community and project at external national and international meetings by giving presentations and running workshops
```

* And a lot more!

(cl-infrastructure-community-managers-skills)=
## What qualifications or skills do you need to be a Research Community Manager? 

> The vast majority of Research Community Managers will have a scientific background that may include advanced degrees (at a Masters or Doctoral level). 

Reviewer: ```suggestion
The vast majority of Research Community Managers will have a scientific background that may include advanced degrees (at a Master's or Doctoral level). 
```

> Many Research Community Managers also have a background related to the specific field or discipline they manage a community in, but not all of them do - many parts of the research process and tools for collaboration are transferable across domains. 

Reviewer 2: ```suggestion
Many Research Community Managers also have a background related to the specific field or discipline they manage a community in, but not all of them do - many parts of the research process, tools and practices for collaboration are transferable across domains. 
```

> If the community is associated with software or programming, it is common for community managers to have some coding skills. 

Reviewer 2: ```suggestion
If the community is associated with software or programming, it is common for Research Community Managers to have some coding skills. 
```

> The focus on inclusive collaboration requires that Research Community Managers have significant depth of understanding in issues relating to Equality, Diversity, Inclusion and Belonging <mark>link to glossary?</mark>. 

Reviewer: Or Equity?

There is no professionally recognised qualification or training course to become a Research Community Manager, but organisations do offer training and resources to help support the professional development of people in these roles.

> The values and approaches community managers bring to their roles are often the most important qualifications for success as a community manager.

Reviewer 2: ```suggestion
The values and approaches Research Community Managers bring to their roles are often the most important qualifications for success as a Research Community Manager.
```

## What do Research Community Managers do? 

Research Community Managers work to engage, enthuse and organise groups of scientists, researchers and/or patients and the public around shared research topics and objectives. 
They are often involved in culture change to support collaboration between members of the community, through the development of community leadership and agency. 
They value the people doing the research and develop structures to support the generation of bottom-up activity <mark>does bottom-up need a definition/glossary listing?</mark>. Their goals are to:

* Embed open, inclusive and reproducible research practices
* Ensure a shared understanding of goals, roadmap, processes
* Facilitate stakeholder engagement and collaboration
* Provide technical support and domain expertise
* Co-create, maintain and communicate project resources
* Amplify and champion community learnings and achievements

Each role is varied but the main activities are typically focused around advocacy for the community, stewardship, scaffolding for collaboration, and the flow of information across all levels of the research, both to and from leadership, and collaboration with the community. 

> Effective collaboration can also require management of relationships between colleagues, including coaching, formal and informal mediation, reflexive exercises, and dealing with formal Code of Conduct <mark>link to glossary?</mark> processes. 

Reviewer 2: ```suggestion
Effective collaboration can also require management of relationships between colleagues, including coaching, formal and informal mediation, reflexive exercises, and dealing with formal [{term}`def<Code of Conduct>`] processes. 
```

* Planning for upcoming community initiatives
* Running training courses and workshops
* Reading and sharing publications relevant to the community
* Managing membership lists
* Mentoring community contributors
* Meeting with and listening to community members to understand their needs and challenges
* Maintaining stakeholder and project databases
* Advocating for the community and project at external national and international meetings by giving presentations and running workshops
* And a lot more!

(cl-infrastructure-community-managers-skills)=
## What qualifications or skills do you need to be a Research Community Manager? 
The vast majority of Research Community Managers will have a scientific background that may include advanced degrees (at a Master's or Doctoral level). 
Many Research Community Managers also have a background related to the specific field or discipline they manage a community in, but not all of them do - many parts of the research process, tools and practices for collaboration are transferable across domains. 
If the community is associated with software or programming, it is common for Research Community Managers to have some coding skills. 

> The focus on inclusive collaboration requires that Research Community Managers have significant depth of understanding in issues relating to Equality, Diversity, Inclusion and Belonging <mark>link to glossary?</mark>. 

Reviewer 2: ```suggestion
The focus on inclusive collaboration requires that Research Community Managers have significant depth of understanding in issues relating to Equality, Diversity, Inclusion and Belonging - see [{term}`def<Equitable, Diverse and Inclusive Practices>`] . 
```

The values and approaches Research Community Managers bring to their roles are often the most important qualifications for success as a Research Community Manager.

```{admonition} Community Skills and Core Competancies 
The Community Skills Framework™ by Community Roundtable includes five skill families: Content, Technical, Business, Engagement and Strategic community management skills.
See the full document [online](https://communityroundtable.com/what-we-do/models-and-frameworks/community-skills-framework/) under the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License. 

The CSCCE Skills Wheel include five core competancies: technical skills, communication, programme management, programme development and interpersonal.  
See the full document on [Zenodo](https://zenodo.org/record/4437294#.YYjzg07P1aQ) under the Creative Commons Attribution Non Commercial No Derivatives 4.0 International.
RCM Skills Framework from Turing Research Community Management Team

> Sharan, M., & Karoune, E. (2023). Research Community Management - Skills framework. Zenodo. https://doi.org/10.5281/zenodo.8337627

Reviewer 2: ```suggestion
Sharan, M., & Karoune, E. (2023). Research Community Management - Skills framework. Zenodo. https://doi.org/10.5281/zenodo.8337627
```
```

The values and approaches Research Community Managers bring to their roles are often the most important qualifications for success as a Research Community Manager.

```{admonition} Community Skills and Core Competancies 
The Community Skills Framework™ by Community Roundtable includes five skill families: Content, Technical, Business, Engagement and Strategic community management skills.
See the full document [online](https://communityroundtable.com/what-we-do/models-and-frameworks/community-skills-framework/) under the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License. 

The CSCCE Skills Wheel include five core competancies: technical skills, communication, programme management, programme development and interpersonal.  
See the full document on [Zenodo](https://zenodo.org/record/4437294#.YYjzg07P1aQ) under the Creative Commons Attribution Non Commercial No Derivatives 4.0 International.
RCM Skills Framework from Turing Research Community Management Team
Sharan, M., & Karoune, E. (2023). Research Community Management - Skills framework. Zenodo. https://doi.org/10.5281/zenodo.8337627

> 

Reviewer 2: ```suggestion
```
```

```

(cl-infrastructure-community-managers-tasks)=
## What do Research Community Managers do? 

Research Community Managers work to engage, enthuse and organise groups of scientists, researchers and/or patients and the public around shared research topics and objectives. 
They are often involved in culture change to support collaboration between members of the community, through the development of community leadership and agency. 
They value the people doing the research and develop structures to support the generation of bottom-up activity. Their goals are to:

* Embed open, inclusive and reproducible research practices
* Ensure a shared understanding of goals, roadmap, processes
* Facilitate stakeholder engagement and collaboration
* Provide technical support and domain expertise
* Co-create, maintain and communicate project resources
* Amplify and champion community learnings and achievements

Each role is varied but the main activities are typically focused around advocacy for the community, stewardship, scaffolding for collaboration, and the flow of information across all levels of the research, both to and from leadership, and collaboration with the community. 

> Effective collaboration can also require management of relationships between colleagues, including coaching, formal and informal mediation, reflexive exercises, and dealing with formal [{term}`def<Code of Conduct>`] processes. 

Reviewer 2: ```suggestion
Effective collaboration can also require management of relationships between colleagues, including coaching, formal and informal mediation, reflexive exercises, and dealing with formal Code of Conduct [{term}`def<Code of Conduct>`] processes. 
```

There is no professionally recognised qualification or training course to become a Research Community Manager, but organisations do offer training and resources to help support the professional development of people in these roles.
The values and approaches Research Community Managers bring to their roles are often the most important qualifications for success as a Research Community Manager.

```{admonition} Community Skills and Core Competancies 
The Community Skills Framework™ by Community Roundtable includes five skill families: Content, Technical, Business, Engagement and Strategic community management skills.
See the full document [online](https://communityroundtable.com/what-we-do/models-and-frameworks/community-skills-framework/) under the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License. 

The CSCCE Skills Wheel include five core competancies: technical skills, communication, programme management, programme development and interpersonal.  
See the full document on [Zenodo](https://zenodo.org/record/4437294#.YYjzg07P1aQ) under the Creative Commons Attribution Non Commercial No Derivatives 4.0 International.

> RCM Skills Framework from Turing Research Community Management Team

Reviewer 2: ```suggestion
RCM Skills Framework from Turing Research Community Management Team - 
```

See the full document on [Zenodo](https://zenodo.org/record/4437294#.YYjzg07P1aQ) under the Creative Commons Attribution Non Commercial No Derivatives 4.0 International.
```

<mark>Add more here from the TPR RCM paper on skills</mark>

(cl-infrastructure-community-managers-challenges)=
## Challenges for Research Community Managers 

> * Need to mediate between community members 

Reviewer: Suggestion to replace the word 'need' throughout this section with 'required to'

> * May not be seen as a part of the community themselves and need to build trust and credibility within it

Reviewer: ```suggestion
* May not be seen as part of the community themselves and need to build trust and credibility within it
```

> * Need to have a lot of a broad range of skills and expertise (technical, interpersonal, project management)

Reviewer 2: ```suggestion
* Need to have a broad range of skills and expertise (technical, interpersonal, project management)
```

* Supporting and encouraging engagement in the community 
* Building infrastructure from scratch in newly created roles 

> * Being seen as a professional in their own right and not just adminstrative staff 

Reviewer: ```suggestion
* Being seen as a professional in their own right 
```
- 'not just admin staff' could be seen as condescending towards admin staff

* Connecting with new audiences who are not aware of the community
* Bridging and translating between different groups in the same field or institution 

> * Managing tasks where there is little formal process in place 

Reviewer: ```suggestion
* Managing tasks where there are little formal processes in place 
```

> * Managing the different priorities of different stakeholder groups 

Reviewer: Managing priorities between different stakeholder groups

* Not always visible when things go well!
* Lack of formal career progression

(cl-infrastructure-community-managers-benefits)=
## Benefits to having Research Community Managers
* Able to offer meta-thinking about how the community is structured and run 

> * Shares best practices around communication, collaboration, diversity, equity and inclusion, and other areas of research 

Reviewer 2: ```suggestion
* Shares best practices around research, communication, collaboration, diversity, equity and inclusion, and other areas of research 
```

* Supports the upskilling of members via technical skill sharing and training
* Supports other members of the community to take on more active roles, including leadership, increasing sustainability, resilience and expansion of the community
* Stewarding initiatives to develop the community such as data standards, a Code of Conduct <mark>check style sheet on capitalisation of this</mark>, or training workshops

> * Offers a stable base for the community, to make information is captured, meetings happen and relationships are strengthened

Reviewer 2: ```suggestion
* Offers a stable base for the community, to make sure information is documented consistently, meetings happen and relationships are strengthened
```

> * Offers a stable base for the community, to make information is captured, meetings happen and relationships are strengthened

Reviewer: ```suggestion
* Offers a stable base for the community, to make sure information is captured, meetings happen and relationships are strengthened
```

> * Connecting groups working on similar projects together, to support increased collaboration 

Reviewer: ```suggestion
* Connecting groups working on similar projects to support increased collaboration 
```

> * Breaking down siloes between departments, fields, research groups 

Reviewer: ```suggestion
* Breaking down silos between departments, fields and research groups 
```

See the full document on [Zenodo](https://zenodo.org/record/4437294#.YYjzg07P1aQ) under the Creative Commons Attribution Non Commercial No Derivatives 4.0 International.
```

<mark>Add more here from the TPR RCM paper on skills</mark>

(cl-infrastructure-community-managers-challenges)=
## Challenges for Research Community Managers 
* Need to mediate between community members 
* May not be seen as part of the community themselves and need to build trust and credibility within it
* Need to have a broad range of skills and expertise (technical, interpersonal, project management)
* Supporting and encouraging engagement in the community 
* Building infrastructure from scratch in newly created roles 
* Being seen as a professional in their own right 
* Connecting with new audiences who are not aware of the community
* Bridging and translating between different groups in the same field or institution 
* Managing tasks where there is little formal process in place 

> * Managing the different priorities of different stakeholder groups 

Reviewer: ```suggestion
* Managing priorities between different stakeholder groups 
```

See the full document on [Zenodo](https://zenodo.org/record/4437294#.YYjzg07P1aQ) under the Creative Commons Attribution Non Commercial No Derivatives 4.0 International.
```

> <mark>Add more here from the TPR RCM paper on skills</mark>

Reviewer 2: ```
```

> <mark>Add more here from the TPR RCM paper on skills</mark>

Reviewer 2: ```suggestion

```
